ci/pipeline.yml

```diff
@@ -224,6 +224,7 @@ jobs:
         params:
           GOPROXY: ((repository_mirrors.goproxy))
           GOSUMDB: ((repository_mirrors.gosumdb))
+          GOOS_LIST: linux,windows
```

Comment:
i assume this is a task specific thing?

Reply:
Yup: https://github.com/cloudfoundry/bosh-package-golang-release/blob/main/ci/tasks/shared/bump-deps.sh#L27